website/docs/docs/jira.mdx

```diff
@@ -0,0 +1,223 @@
+---
+sidebar_position: 3
+slug: /jira-sync
+---
+
+# Jira sync
+
+The Jira sync binary runs one cycle and then exits.
+For automatical sync schedule executions of the module with cron or similar.
+Currently supported are titles, desriptions, comments and labels
+
+## Single Execution
+
+Each cycle consists of two parts: Incoming and Outgoing.
+The incoming phase only syncs data from Jira to gropius while the outgoing phase syncs only from gropius to Jira.
+
+### Incoming
+
+During the Incoming phase the sync first downloads all new events from Jira.
+These events are immediately added to the MongoDB and the timestamps for the Jira api moved to avoid repeated downloads.
+
+After everything has been downloaded, the missing issues are created and all timeline items inserted into the timelines.
+
+### Outgoing
+
+The outgoing phase collects all new issues and timeline items that have no known Jira id and have not been invalidated (e.g. no reopen after close event).
+These differences are then collected and prepared for submission to Jira.
+
+After all mutations are collected, a final damage control check ensures that the amount of mutations does not exceed a threshold to prevent damage to larger public repositories in case of issues with the sync.
+Then all of the mutations are executed and the ids of the created issues and timeline items added to the MongoDB.
+
+## How to use
+
+When the server side is running, the repos which should be synced have to be set.
+To mark an `IMS` to connect to Jira, the sync service creates the Jira `IMSTemplate` on first run.
+
+This can be used to create an `IMS`, which requires:
+* the id of an `IMSUser` owning the token which is then used to query the issues (`null` until an `IMSUser` is created for this `IMS`)
+* and the URL of the GraphQL endpoint of the Jira instance (e.g. `https://api.jira.com/graphql` for the Jira Jira)
```

Comment:
why do I have the feeling you did a bit to much copy paste search and replace in this file?